azerothcore/azerothcore-wotlk
# Fix(Core): GM command not working while GM mode is on

##### CHANGES PROPOSED:

-  Call the correct function for int32 value in prepared statement query


###### ISSUES ADDRESSED:


Closes #1023 


##### TESTS PERFORMED:


Tested in game, compiled on win 10 at 64bit, no errors

##### HOW TO TEST THE CHANGES:

Can test in the same way of issue 1023, but is possible test near any gob or npc. I reccomend to try with gob and npc with different phaseMask.

STEPS TO TEST THE FIXED PROBLEM (GOB):
    .tele orgrimmar
    .gm on
    .gob near
    before fix doesn't work, nob gob found instead 9 (now work)
    .gm off
    .gob near
    works
 
STEPS TO TEST THE FIXED PROBLEM (NPC):

- Under the building in same position, there are some npc in other phaseMask, you can see in GM ON, before fix the command .npc near doesn't work, now with the same command you can see the npc position

##### KNOWN ISSUES AND TODO LIST:


none


##### Target branch(es):

Master
